website_animate/static/src/js/o_animate.editor.js

```diff
@@ -51,7 +32,15 @@ sOptions.registry.o_animate = sOptions.Class.extend({
     selectClass: function (previewMode, value, $opt) {
         this._super.apply(this, arguments);
         forceAnimation(this.$target);
-        this.$target.toggleClass('o_animate_preview o_animate', !!value);
+        this.$target.toggleClass('o_animate o_animate_preview', !!value);
+    },
+    /**
+     * @override
+     */
+     destroy: function () {
+        this._super.apply(this, arguments);
+        // Clean elements
+        this.$target.removeClass('o_animate_preview');
```

Comment:
You cannot modify the target in the `destroy` of an option, why not in `cleanForSave` (which is normally called around the same time as `destroy` anyway)?

Reply:
Indeed, Actually I have to remove `this._super.apply(this, arguments);`  from the cleanForSave to fix the bug